docs/reference/metadata.md

```diff
@@ -0,0 +1,1273 @@
+| ``given_name``  | (0-1) | Given name(s) if the type is a personal name. |
+| ``family_name`` | (0-1) | Family name if type is a personal name. |
+| ``identifiers``  | (0-n) | Person or organisation identifiers. |
+| ``affiliations`` | (0-n) | Affilations if type is a personal name. |
+
+Affiliations are described with the following subfields:
+
+| Field | Cardinality |   Description   |
+|:-----:|:-----------:|:----------------|
+| ``name`` | (1) | The organizational or institutional affiliation of the creator. |
+| ``identifiers`` | (0-n) | Organisation identifiers. |
+
+Identifiers are described with the following subfields (note, we only support one identifier per scheme):
+
+| Field | Cardinality |   Description   |
+|:-----:|:-----------:|:----------------|
+| ``scheme`` | (1, CV) | The identifier scheme. |
+| ``value`` | (1) | Actual value of the identifier. |
+
+Supported creator identifier schemes:
+
+- ORCID
+- GND
+- ISNI
+- ROR
+
+Supported affiliation identifier schemes:
+
+- ISNI
+- ROR
+
+Example:
+
+```json
+{
+    "creators": [{
+      "name": "Nielsen, Lars Holm",
+      "role": "creator",
+      "type": "personal",
+      "given_name": "Lars Holm",
+      "family_name": "Nielsen",
+      "identifiers": {
+        "orcid": "0000-0001-8135-3489",
+        "<scheme>": "<value>"
+      },
+      "affiliations": [{
+        "name": "CERN",
+        "identifiers": {
+          "ror": "01ggx4157",
+          "isni": "000000012156142X",
+          "<scheme>": "<value>"
+        }
+      }]
+    }],
+}
+```
+
+Note that the identifier's schemes are lowercased.
+
+### Title (1)
+
+A primary name or primary title by which a resource is known. May be the title of a dataset or the name of a piece of software. The primary title is used for display purposes throughout InvenioRDM.
+
+The fields is compatible with *3. Title* in DataCite. Compared to DataCite the field does not support specifying the language of the title.
+
+Example:
+
+```json
+{
+    "title": "InvenioRDM"
+}
+```
+
+### Additional titles (0-n)
+
+Additional names or titles by which a resource is known.
+
+The fields is compatible with *3. Title* in DataCite. Compared to DataCite, InvenioRDM splits *3. Title* into two separate fields ``title`` and ``additional_titles``. This is to ensure consistent usage of a record's title throughout the entire software stack for display purposes.
+
+Subfields:
+
+- ``title`` (1) - The addtional title.
+- ``type`` (1, CV) - The type of the title. One of: ``alternative_title``, ``subtitle`` ``translated_title`` or ``other``.
+Other
+- ``lang`` (0-1, CV) - The language of the title. ISO 639-3 language code.
+
+Example:
+
+```json
+{
+    "additional_titles": [{
+      "title": "a research data management platform",
+      "type": "subtitle",
+      "lang": "eng"
+    }]
+}
+```
+
+### Description (1)
+
+The description of a record.
```

Comment:
Isn't the same case then with title? description is also split in two and does not support type (which datacite has). In addition, additional_descriptions adds `lang` (which IIRC datacite does not have.)